.circleci/config.yml

```diff
@@ -36,6 +36,7 @@ jobs:
     steps:
       - checkout
       - run: cp -p config/secrets_example.yml config/secrets.yml
+      - run: gem install bundler:2.2.15
```

Comment:
This line shouldn't be necessary. I14Y is using [CircleCi's Ruby orb](https://circleci.com/developer/orbs/orb/circleci/ruby) (`circleci/ruby@1.1.2`), which provides us with automagic goodies including bundler installation (via the `ruby/install-deps` step). "By default, it gets the bundler version from Gemfile.lock." If that doesn't work for some reason, we can configure it via the `bundler-version` parameter:
https://circleci.com/developer/orbs/orb/circleci/ruby#commands-install-deps

I realized I led you astray by specifying that we needed to bump the version in this file. The "orb" setup is pretty new (and isn't yet installed in all our CircleCi configs), so I didn't realize we could skip this step going forward. I'll update Jira/Confluence accordingly.

Reply:
Removed.